pandas/tests/groupby/test_transform.py

```diff
@@ -498,6 +498,30 @@ def test_cython_transform_series(self, op, args, targop):
             tm.assert_series_equal(expected, getattr(
                 data.groupby(labels), op)(*args))
 
+    @pytest.mark.parametrize("op", ['cumprod', 'cumsum'])
+    @pytest.mark.parametrize("kwargs", [{'skipna': False}, {'skipna': True}])
```

Comment:
you can just rename this to skipna and make it take [False, True].

Reply:
Good point, did that.